Dockerfile

```diff
@@ -15,21 +15,26 @@
 # This is a multi-staged Dockerfile that uses Maven builder image to build the whole project with Maven.
 # In the second phase, the standalone Quarkus executable JAR is placed into an OpenJDK JRE image.
 #
-# Build an image:
+# Build an image (that uses in-memory H2 database):
 # docker build -t optaweb/employee-rostering --ulimit nofile=98304:98304 .
 #
-# Run the image with default profile (using in-memory H2 database):
+# Build an image (that uses PostgreSQL database):
+# docker build -t optaweb/employee-rostering --build-arg QUARKUS_PROFILE=postgres --ulimit nofile=98304:98304 .
+#
+# Run the image (if built for in-memory H2 database):
 # docker run -p 8080:8080 --rm -it optaweb/employee-rostering
 #
-# Run the image with production profile (using PostgreSQL database):
-# docker run -p 8080:8080 --rm -it -e QUARKUS_PROFILE=production optaweb/employee-rostering
+# Run the image (if build for PostgreSQL database):
+# docker run -p 8080:8080 --rm -it -e DATABASE_NAME=$NAME -e DATABASE_USER=$USER -e DATABASE_PASSWORD=$PASSWORD optaweb/employee-rostering
```

Comment:
Not sure where this information comes from. Is the user expected to run an external database somewhere?
The instructions don't really say.

Reply:
As far as I can tell, the postgres profile is explicitly for an external database. Passing those variables with the added build arguments did work for my build.

Reply:
It for an external database deployed on the local machine (either from a different docker image, setup on local machine, etc.). If a cloud provider/outside database is used w/o URL remapping for `postgres`, the environment variable `DATABASE_HOST` would also need to be set to the database location.

Reply:
@Christopher-Chianelli I'd say that deliberately in the comments. `DATABASE_HOST` is not mentioned there at all, for example. If we're using the comments as a howto, let's describe the use cases.